Assets/GothicVR/Scripts/UI/TextureManager.cs

```diff
@@ -53,9 +53,8 @@ public void LoadCustomTextures()
         arrowtexture = AssetCache.I.TryGetTexture("U.TGA");
         arrowmaterial.mainTexture = arrowtexture;
 
-        // FIXME - No material found in Assets. commenting it out right now.
-        // skytexture = AssetCache.I.TryGetTexture("SKYDAY_LAYER0_A0-C.TEX");
-        // skymaterial.mainTexture = skytexture;
+        skytexture = AssetCache.I.TryGetTexture("SKYDAY_LAYER0_A0-C.TEX");
```

Comment:
PS: You don't need to add -C.TEX or .TGA. The TryGetTexture() method is removing the suffix and will test it on it's own.
No bug, just something I wanted to mention. :-)

Reply:
Ok. Cool. I will change it.